cg/cli/demultiplex/demux.py

```diff
@@ -178,6 +179,7 @@ def confirm_flow_cell_sync(context: CGConfig, source_directory: str):
     help="The path from where the syncing is done.",
 )
 def create_manifest_files(source_directory: str):
-    for source_flow_cell in Path(source_directory).iterdir():
-        if is_manifest_file_required(source_flow_cell):
-            create_manifest_file(source_flow_cell)
+    """Creates a file manifest for each flow cell in the source directory."""
+    for source_flow_cell in glob(f"{source_directory}/*"):

```

Comment:
```suggestion
    for source_flow_cell in get_directories_in_path(Path: Path):

```
does already exist for this

Reply:
Can't use wildcards in it unfortunately 😢